building/javascript-sdk.md

```diff
@@ -234,7 +234,12 @@ Since all reads are always open to the public, and Tableland is still in very ea
 
 ### Listing Tables
 
-Once tables have been created for a given address, they can be listed via the `list` function. This function takes no arguments and returns a list of `TableMetadata` objects for the chain that was connected to. `TableMetadata` objects contain `controller`, `name`, and `structure`. The `structure` of a table is defined by its normalized schema, more on `structure` in the section on [Schema Structure](#schema-structure).
+Once tables have been created for a given address, they can be listed via the `list` function. This function takes no arguments and returns a list of `TableMetadata` objects for the chain that was connected to.
+`TableMetadata` objects contain:
+  - `controller`, the checksummed address of the account that owns the table
```

Comment:
I was tripped up by the fact that this is a checksummed address, and figured we might as well document it so others don't have the same issue